docs/community/working_groups.md

```diff
@@ -0,0 +1,38 @@
+# Working groups 
+
+Working groups are subteams of the napari community formed by members interested in diving deep on a particular topic (ex:‘Bundled Application’, ‘Documentation’) that meet on a semi-regular cadence to discuss, implement, and track ideas over time. 
+
+## Working group guidelines
+
+We believe working groups are helpful ways of moving forward work on particular topics, and the following guidelines serve to help team members form and/or maintain healthy working groups. Working groups should:
+
+- Identify at least two co-leads responsible for the creation and continuation of the group. It may be helpful to have one [core developer](https://napari.org/community/team.html#current-core-developers) “sponsor” who is kept aware of work and can help shepherd through PRs as needed. All other membership is opt-in.
+- Create a “charter” that describes working group’s core focus and goals, holds meeting minutes, lists membership/zulip channel/meeting schedule. 
+- Identify achievable goals and deliverables that map thoughtfully to napari’s larger strategies and [roadmap](https://napari.org/roadmaps/index.html). These goals may focus on specific short-term goals or longer term efforts that still require exploration. 
+- Hold regular public meetings that are listed on the [napari community calendar](https://napari.org/community/team.html#meeting-schedule). Cadence is up to the team, but monthly minimum is recommended.
+- Maintain and publicly store the working group’s meeting minutes in [napari/meeting-notes](https://github.com/napari/meeting-notes).
+- Present updates at least once per month at a napari community meeting (done by at least one working group member).
+- Have members actively working towards goals set forth in the charter.
+- Reach out to and collaborate with other open source communities to bring new expertise to the project (when relevant).
+
+## Existing working groups
+
+_Click on a group to view members, minutes, etc._
```

Comment:
Do these pages exist yet? If not we might want to wait before saying this. Right now I think we can just click on the co-leads and get to their personal githubs, which is different. I do like the idea though!!

Reply:
Yeah, and even if we make github teams, I'm pretty sure no-one can see them unless they're already a member of the napari github organisation - so it's pretty pointless. We've had the same conversation about the docs link to the core maintainers list: no-one can see that unless they are already a member of the napari github org so it isn't very useful.

Reply:
I would remove this line if we don't have the links yet. They can wait for a future PR but having the line here is confusing.